client/lib/plans/constants.js

```diff
@@ -1,28 +1,58 @@
+import i18n from 'lib/mixins/i18n';
+
 export const plansList = {
-	'free_plan': {},
-	'value_bundle': {},
-	'business-bundle': {},
+	'free_plan': {
+		getTitle: () => i18n.translate( 'Free' ),
+		getPriceTitle: () => i18n.translate( 'Free for life' )
+	},
+	'value_bundle': {
+		getTitle: () => i18n.translate( 'Premium' ),
+		getPriceTitle: () => i18n.translate( '$99 per year' )
+	},
+	'business-bundle': {
+		getTitle: () => i18n.translate( 'Business' ),
+		getPriceTitle: () => i18n.translate( '$299 per year' )
+	},
 	'jetpack_free': {},
 	'jetpack_business': {}
 };
 
+const allPaidPlans = [
+	'value_bundle',
+	'business-bundle'
+];
+
 export const featuresList = {
 	'google-analytics': {
+		getTitle: () => i18n.translate( 'Google Analytics' ),
 		plans: [ 'business-bundle' ]
```

Comment:
Let's leave a note in this file that this is only meant to be used for our upsell feature pages. I just want to avoid having folks using this as a feature check elsewhere.